var/spack/repos/builtin/packages/exprtk/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Exprtk(Package):
+    """ExprTk - C++ Mathematical Expression Toolkit Library"""
+
+    homepage = "https://www.partow.net/programming/exprtk/index.html"
+    url = "https://github.com/ArashPartow/exprtk/archive/refs/heads/release.tar.gz"
+    git = "https://github.com/ArashPartow/exprtk.git"
+
+    maintainers = ["arashpartow"]
```

Comment:
Directive preferred.
```suggestion
    maintainers("arashpartow")
```